scss/components/_filters.scss

```diff
@@ -1,10 +1,23 @@
 .filter-box {
-  img {
+  position: relative;
+
+  &::before {
+    content: '';
+    padding-top: 56.25%;
     display: block;
+  }
+
+  > * {
+    position: absolute;
+    top: 0;
+    left: 0;
     width: 100%;
+    height: 100%;
+    object-fit: cover;
```

Comment:
please remove the duplicate from filters box, just use the `ratio-box` class on those divs